examples/opensuse.yaml

```diff
@@ -3,7 +3,10 @@ images:
 # Hint: run `limactl prune` to invalidate the "Current" cache
 - location: https://download.opensuse.org/distribution/leap/15.4/appliances/openSUSE-Leap-15.4-JeOS.x86_64-15.4-OpenStack-Cloud-Current.qcow2
   arch: "x86_64"
-# No aarch64 OpenStack build found :(
+  digest: "sha256:378c51ae8a064513410285ffe884fa455e6325c583861aebe11cff7291298195"
+- location: https://download.opensuse.org/distribution/leap/15.4/appliances/openSUSE-Leap-15.4-ARM-JeOS-efi.aarch64.qcow2
+  arch: "aarch64"
+  digest: "sha256:79d63c9096e13fc5b615c4807ac845b3afe6bed4f4d71ae257f764d4ac2e11fc"
```

Comment:
The digest changes day by day, so it shouldn't be pinned

Reply:
@e-minguez Please drop this digest, and I'm happy to merge this PR.
Migration to Minimal-VM can be discussed later.

Reply:
I don't see the digest changing; it just looks like the digest in the PR is wrong. The correct digest should be:

```
$ cat openSUSE-Leap-15.4-ARM-JeOS-efi.aarch64.qcow2.sha256
965691484a2033f95789050820921ab847d1827a375038831aa7a237718099a5 openSUSE-Leap-15.4-ARM-JeOS-efi.aarch64.qcow2
```